code/build.gradle

```diff
@@ -187,7 +187,7 @@ repositories {
 
 node {
     // Version of node to use.
-    version = '16.15.1'
+    version = '20.12.2'
```

Comment:
Hmm, mener vi har holdt tilbake på node oppgradering for apper i og med at vi fortsatt har prosjekter som kan være på lavere node versjon.

Reply:
Dette kan selvfølgelig testes, men mener @SkylineTardis fortalte meg det en gang jeg oppgraderte pakker og node på en av appene 😊

Reply:
Det bør ikke ha noe å si, node brukes til å innhente pakkene under byggprosessen, og når jar-fila er ferdig bygget, så er den bygget. Jeg har dobbeltsjekket med denne versjonen på Delta, som har lavere node, og den fungerer fortsatt.

Reply:
I asked her, and she thought the same thing. i.e regarding the XP version.